enduro.toml

```diff
@@ -15,12 +15,40 @@ debug = false
 corsOrigin = "http://localhost"
 
 [api.auth]
+# Enable API authentication, only OIDC auth. is available at the moment.
+# The API only verifies the access token submitted on each request when
+# enabled. Obtaining the token is done and configured in the dashboard.
 enabled = true
 
 [api.auth.oidc]
+# OIDC provider URL (required with auth. is enabled). 
 providerURL = "http://keycloak:7470/realms/enduro"
+# OIDC client ID that must be included in the `aud` claim from the access
+# token (required with auth. is enabled). 
 clientID = "enduro"
 
+[api.auth.oidc.abac]
+# Enable Attribute Based Access Control (ABAC). If enabled, the API will
+# check a configurable multivalue claim against required attributes based
+# on each endpoint configuration. If extra scopes are needed to obtain
+# such claim they can be configured in the dashboard.
+enabled = false
+# Claim name/path in the access token where the Enduro attributes should
+# be found. If the claim is nested include all fields with a separator.
+# For example "attributes.enduro", using "." as separator below. Required
+# when ABAC is enabled.
+claimPath = "enduro"
+# Separator used to split the claim path fields. Defaults to "", it will
+# try to find the claim path as it is in the top-level fields.
```

Comment:
Just that if you use "attributes.enduro" as the claim path but don't set a separator, it will try to find a field called exactly like that in the top-level fields from the access token. Maybe `Defaults to "", the claim path will be looked as it is in the top-level fields from the access token.`?

Reply:
Maybe `The default value of "" will try to match the claim path as-is to a top-level field from the access token.`?

Reply:
A lot better, thanks!